plugins/content/attachment/services/AttachmentAssetService.php

```diff
@@ -494,11 +494,11 @@ public function getAction($attachmentAssetId)
 	 * List attachment Assets by filter and pager
 	 * 
 	 * @action list
-	 * @param KalturaAssetFilter $filter
+	 * @param KalturaAttachmentAssetFilter $filter
 	 * @param KalturaFilterPager $pager
 	 * @return KalturaAttachmentAssetListResponse
 	 */
-	function listAction(KalturaAssetFilter $filter = null, KalturaFilterPager $pager = null)
+	function listAction(KalturaAttachmentAssetFilter $filter = null, KalturaFilterPager $pager = null)
```

Comment:
That line exists because compat-check in the past showed that there is a customer that uses KalturaAssetFilter as input. If you'll apply your change you will break the application of that customer, I can't remember who it was.